hexrd/unitcell.py

```diff
@@ -751,7 +751,7 @@ def InitializeInterpTable(self):
                 if Z <= 92:
                     gid = fid.get('/'+elem)
                     data = np.array(gid.get('data'))
-                    self.pe_cs[elem] = interp1d(data[:, 7], data[:, 3])
+                    self.pe_cs[elem] = interp1d(data[:, 7], data[:, 3]+data[:,4])
```

Comment:
It would be nice to have these columns stored as variables to make this easier to read (such as `SCATTERING_FACTORS_IDX` or something like that).